perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -180,21 +211,19 @@ def _ListObjects(storage_schema, bucket, prefix, host_to_connect=None):
     A list of object names.
   """
 
-  bucket_list_result = None
   if _useBotoApi(storage_schema):
     bucket_uri = boto.storage_uri(bucket, storage_schema)
     if host_to_connect is not None:
       bucket_uri.connect(host=host_to_connect)
 
-    bucket_list_result = bucket_uri.list_bucket(prefix=prefix)
+    return [key.name for key in bucket_uri.list_bucket(prefix=prefix)]
+  elif _useBoto3Api(storage_schema):
+    s3 = _getS3()
```

Comment:
where do you implement the .connect() equivalent here in boto3?

The caller of this script passes in "host=host_to_connect" which is a fully qualified DNS name.